server/src/server.js

```diff
@@ -371,6 +352,41 @@ app.use(function (err, req, res, next) {
   errorResponse(res, "General error:", err);
 });
 
+const contentApp = express();
+
+contentApp.use("/static", express.static(path.join(__dirname, "static"), {
+  index: false
+}));
+
+contentApp.use(function (req, res, next) {
```

Comment:
I think this is copied from elsewhere, right?  Could we make it into a named function to be reused?